### DISCUSSION TITLE
Dns-Over-Https

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1920] [General]
This is what www.bromite.org/doh says:

> This feature is disabled by default.

This isn't True since by default Dns-Over-Https is set to this :
![Screenshot_20220323-164300](https://user-images.githubusercontent.com/95953932/159686926-2e25c80c-543a-4cec-9dea-d8982ae6e9bd.jpg)

**Comment by csagan5** [comment]
You are right, that page is outdated because Chromium did not have DoH enabled by default back then.

Ideally we should remove automatic mode altogether but for now I will update the website page.

**Reply by ghost** [reply]
> You are right, that page is outdated because Chromium did not have DoH enabled by default back then.
> 
> Ideally we should remove automatic mode altogether but for now I will update the website page.

Oh okay, Thanks
